.github/workflows/docker.yml

```diff
@@ -0,0 +1,79 @@
+name: Create and publish a Docker images
+
+on:
+  push:
+    tags:
+      - "v*.*.*"
+    branches:
+      - "master"
+
+env:
+  REGISTRY: ghcr.io
+  IMAGE_NAME: ${{ github.repository }}
+
+jobs:
+  build-and-push-image:
+    name: PHP ${{ matrix.php-version }}, Fixer ${{ github.ref == 'refs/heads/master' && 'edge' || github.ref_name }}
+    if: github.repository == 'PHP-CS-Fixer/PHP-CS-Fixer'
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        include:
+          - php-version: '7.4'
```

Comment:
when dropping 7.4 support or bringing 8.4 support, I will for sure forget to get this list updated.

non-blocking idea: extend https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/blob/master/tests/AutoReview/CiConfigurationTest.php#L46 to check new files

Reply:
I like the idea and extended the test.